GVFS/GVFS.Common/Maintenance/PrefetchStep.cs

```diff
@@ -192,7 +193,12 @@ private bool TryGetMaxGoodPrefetchTimestamp(out long maxGoodTimestamp, out strin
                 }
             }
 
-            if (!this.Stopping && firstBadPack != -1)
+            if (this.Stopping)
+            {
+                throw new StoppingException();

```

Comment:
For my own understanding, what would be the downside to not throwing any `StoppingException` here (or below) in this method?

Is the concern that `gvfs unmount` might take a long time if we need to retry deletes?  It looks like we don't launch any git processes from this point on.

Reply:
@wilbaker , yes that was my concern.  

I am not convinced how necessary it is but wanted to stay consistent with the old code.  

We don't need to rush on it now but a more dangerous, but less verbose method of stopping would be to Abort the thread once we kill the GitProcess.

Reply:
Ok thanks, I think it makes sense to stay consistent with how the old code was checking.